README.md

```
@@ -4,16 +4,4 @@
-> [!IMPORTANT]
-> 프로젝트 내에는 종류 불문하고 어떠한 비밀번호도 절대 평문으로 저장하지 마세요!
->
-> public repository에 키 잘못올리면 골치아파집니다!
-> 
-> `.env` 파일은 `.gitignore`에 추가하여 Git에 커밋되지 않도록 하세요!
-
-## 📅 제출
-
-- **제출 마감**: 2026.01.19 수업 시작 전
-- **제출 방법**: 본 저장소를 Fork하여 작업 후 본인 github id의 브랜치에 PR 제출 할 것
+수행 방법에 대한 상세는 노션 페이지 [4주차 과제](https://www.notion.so/ej31/4-2e3954dac75580e9a969e082171fbd59) 에서 확인해주세요!
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)